src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private final ColorSupplier colorSupplier = new ColorSupplier();
+
+    public Ball getRandomBall() {
+        return new Ball(new Random().nextInt(101), colorSupplier.getRandomColor());
```

Comment:
create a global Random field and use it here, so you won't need to create its instance each time when you call this method